test/commands/distribute/releases/show-test.ts

```diff
@@ -44,22 +44,45 @@ describe("releases show command", () => {
     Nock.enableNetConnect();
   });
 
-  describe("when everything works as expected", () => {
+  describe.only("when everything works as expected", () => {
```

Comment:
We should remove this `.only()`, otherwise we'll disable all other tests in the test suite.

Reply:
Good catch!